decidim-meetings/app/events/decidim/meetings/meeting_event.rb

```diff
@@ -0,0 +1,45 @@
+# frozen-string_literal: true
+
+module Decidim
+  module Meetings
+    # This module is used to be included in events triggered by comments.
+    #
+    module MeetingEvent
+      extend ActiveSupport::Concern
+
+      included do
+        def resource_text
+          translated_attribute(resource.description)
+        end
+
+        def perform_translation?
```

Comment:
These methods (`perform_translation?, content_in_same_language?, and translation_missing`) seem copypasted. Can we move this to a concern? Something like `MachineTranslatedEvent` or whatever naming that you prefer.